File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.cc
#include<stdio.h>
int main(){
	int C,num,sum=0,std=0;
	float average;
	int score[1000];
	scanf("%d",&C); //테스트 케이스 수 
	
	while(C--){
		scanf("%d",&num); //학생 수 
		
		for(int i=0;i<num;i++){
			scanf("%d",&score[i]);
			sum+=score[i];
		}
		
		average = sum / (float)num;
		
		for(int i=0;i<num;i++) if(score[i]>average) std++;
		
		printf("%.3f%%\n",(std/(float)num)*100);
		
		sum=0,std=0; //초기화 
	}
}

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
    	BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    	BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
    	
    	// 테스트 케이스 수 C 입력
    	int C = Integer.valueOf(br.readLine());
    	
    	while(C--!=0) {
    		
    		// 학생수 N, N명의 총점 입력
    		StringTokenizer st = new StringTokenizer(br.readLine());
    		int N = Integer.valueOf(st.nextToken());
    		int sum=0;
    		int[] score = new int[N];
    		
    		for(int i=0;i<N;i++) {
    			score[i]=Integer.valueOf(st.nextToken());
    			sum+=score[i];
    		}
    		
    		// 평균을 넘는 학생의 수 계산
    		int stuNum=0;
    		
    		for(int i=0;i<N;i++)
    			if(score[i]>sum/(double)N)
    				stuNum++;
    		
    		// 평균을 넘는 학생의 비율 출력
    		bw.write(String.format("%.3f", stuNum/(double)N*100)+"%\n");
    	}
    	
    	bw.close();
    } // end of main
} // end of Main class